games/m.yaml

```
@@ -1156,6 +1156,32 @@
   - https://www.artsoft.org/mirrormagic/screenshots/infoscreen.png
   - https://www.artsoft.org/mirrormagic/screenshots/editor.png
 
+- name: Miss Driller
```

```suggestion
- name: Mr. Drillux
```
confusingly the game was ported a few times with name changes each time